src/simulation.hpp

```diff
@@ -316,7 +355,7 @@ template <typename problem_t> class AMRSimulation : public amrex::AmrCore
 
 	// Nghost = number of ghost cells for each array
 	int nghost_cc_ = 4; // PPM needs nghost >= 3, PPM+flattening needs nghost >= 4
-	int nghost_fc_ = 4;
```

Comment:
How does this change interact with MHD? Presumably MHD needs 4, so this number should depend on physics_traits.

Reply:
I'll change this so it uses 4 if MHD is enabled.